docs/guides/walletconnect/walletconnect-user-guide.mdx

```diff
@@ -130,30 +130,30 @@ If you would like to connect your Chia reference wallet to a different DApp, the
   warning ..\..\..\..\package.json: No license field
   $ next start
   ready - started server on 0.0.0.0:3000, url: http://localhost:3000
-  info  - Loaded env from C:\Users\User\Documents\GitHub\chia\chia-wallet-connect-dapp-test\.env.local
+  info  - Loaded env from C:\Users\User\Documents\GitHub\chia\chia-wallet-connect-dApp-test\.env.local
```

Comment:
I was amazed that `git` handles this correctly. On MacOS you'll get a `chia-wallet-connect-dApp-test` directory regardless of the case in GitHub